Assets/Mirage/Runtime/ClientObjectManager.cs

```diff
@@ -85,6 +106,11 @@ void OnClientDisconnected(ClientStoppedReason reason)
             ClearSpawners();
             DestroyAllClientObjects();
             syncVarReceiver = null;
+            _objectPoolingManager?._objectsAssetIds.Clear();
+
+            // Destroy all game objects under parent.
+            for (int child = 0; child < gameObject.transform.childCount; child++)
```

Comment:
we should create a new gameobject to hold the prefab, just incase someone adds child objects to their networkmanager

Reply:
Sure we can do that.